src/test/regress/expected/multi_sequence_default.out

```diff
@@ -605,10 +611,10 @@ SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
 ALTER TABLE sequence_default_8.seq_8 SET SCHEMA sequence_default;
 \c - - - :worker_1_port
 \d sequence_default.seq_8
-                    Sequence "sequence_default.seq_8"
-  Type  |   Start   |  Minimum  |  Maximum  | Increment | Cycles? | Cache
+                     Sequence "sequence_default.seq_8"
+  Type   |   Start   |  Minimum  |  Maximum  | Increment | Cycles? | Cache
 ---------------------------------------------------------------------
```

Comment:
I was surprised to see that this has changed to `integer`, as the default is `bigint` and we have not specified `type` in the `create sequence command`. What I realize is that `create distributed table` changes the default type of the sequence from `bigint` -> `int`. Do you know why is this happening? This seemed weird to me. Do you know what is going on here?

```SQL


CREATE SEQUENCE seq_8 INCREMENT BY 3 CACHE 10 CYCLE;
 \d seq_8 
                           Sequence "public.seq_8"
  Type  | Start | Minimum |       Maximum       | Increment | Cycles? | Cache 
--------+-------+---------+---------------------+-----------+---------+-------
 bigint |     1 |       1 | 9223372036854775807 |         3 | yes     |    10

 CREATE TABLE seq_test_8 (x int, y int DEFAULT nextval('seq_8'));
 \d seq_8 
                           Sequence "public.seq_8"
  Type  | Start | Minimum |       Maximum       | Increment | Cycles? | Cache 
--------+-------+---------+---------------------+-----------+---------+-------
 bigint |     1 |       1 | 9223372036854775807 |         3 | yes     |    10

SELECT create_distributed_table('seq_test_8', 'x');

 create_distributed_table 
--------------------------
 
(1 row)

Time: 38.891 ms

\d seq_8 
                       Sequence "public.seq_8"
  Type   | Start | Minimum |  Maximum   | Increment | Cycles? | Cache 
---------+-------+---------+------------+-----------+---------+-------
 integer |     1 |       1 | 2147483647 |         3 | yes     |    10

Reply:
It was added in this commit of this PR: [`6981fd3` (#5061)](https://github.com/citusdata/citus/pull/5061/commits/6981fd3240f6086f1e7ace0a69c9230524faafe3)